README.md

```diff
@@ -1,9 +1,22 @@
 # vapoursynth
-*A video processing framework with simplicity in mind*
+[![GitHub release](https://img.shields.io/github/release/vapoursynth/vapoursynth?label=Latest%20release&logo=github)](https://github.com/vapoursynth/vapoursynth/releases) [![GitHub](https://img.shields.io/github/license/youka/vapoursynth?label=License&logo=github)](https://github.com/Youka/vapoursynth/blob/master/LICENSE-LGPL-2.1) [![Website](https://img.shields.io/website/http/www.vapoursynth.com?label=Homepage&logo=wordpress)](http://www.vapoursynth.com)  
+[![Travis (.org)](https://img.shields.io/travis/Youka/vapoursynth?label=Linux%20build&logo=travis)](https://travis-ci.org/Youka/vapoursynth) [![AppVeyor](https://img.shields.io/appveyor/ci/Youka/vapoursynth?label=Windows%20build&logo=appveyor)](https://ci.appveyor.com/project/Youka/vapoursynth)
 
-**Looking for the documentation?**  
-Read the documentation here: [VapourSynth Documentation](http://vapoursynth.com/doc)
+*A video processing framework with simplicity in mind.*
+
+## Getting started
+First [install](http://www.vapoursynth.com/doc/installation.html) or [build](#building) Vapoursynth on your system.  
+Then begin with your [first script](http://www.vapoursynth.com/doc/gettingstarted.html).  
+For convenience use an [editor](http://www.vapoursynth.com/doc/pluginlist.html#applications) (like *vsedit* or *Yuuno*).
+
+## Documentation
+* Whole documentation can be built by [source](#building).
+* An online version is [available on homepage](http://vapoursynth.com/doc/).
 
 ## Building
-On **Windows**, follow the build instructions provided in [`build instructions windows.md`](https://github.com/vapoursynth/vapoursynth/blob/master/build%20instructions%20windows.md)  
-On **Linux** and **Mac OS**, follow the build instructions provided in [Linux and OS X Compilation Instructions](http://vapoursynth.com/doc/installation.html#linux-and-os-x-compilation-instructions)
+* **Windows** instructions: [`build_instructions_windows.md`](https://github.com/Youka/vapoursynth/blob/master/instructions/build_instructions_windows.md).  
```

Comment:
Using a relative link here makes more sense. `[build_instructions_windows.md](/instructions/build_instructions_windows.md)`

Reply:
I used absolute paths on purpose for an independent presentation of this project. It may occur that this README markdown document gets embedded somewhere and relative links redirect into nowhere. 

[Example](https://crates.io/crates/ssb_parser) (just click **Changes** link)

Reply:
Yes but still, you are trying to merge it into this repo so I suggest changing the url.